content/4.validators/1.running-a-node/2.install.md

```diff
@@ -33,16 +33,16 @@ git checkout <version-tag>
 ::
 
 ::alert{variant="info"}
-For connecting to the Constantine Developer Testnet, the version tag `v0.2.0` should be used:
+For connecting to the Constantine Developer Testnet, the version tag *v0.2.0* should be used:
 
-`git checkout v0.2.0`
+`git checkout v0.2.0`**`
```

Comment:
This is no longer true. They should be using the latest release published.

Reply:
i have updated now to the latest release